README.md

```diff
@@ -198,17 +205,17 @@ already implemented one that matches your need.
 
 5) **MetricHandler**:
```

Comment:
`prediction outputs` -> `model predictions`
`This class is responsible for postprocess operations applied to prediction outputs.` -> This is confusing since the method does that is named `preprocess`. Moreover, as in my previous comment, we added `postprocess` method that does optional postprocessing to the scores. Therefore, this paragraph should also be updated.